### ISSUE TITLE
New tab becomes blank after pasting address and pressing enter

### ISSUE THREAD

**Issue by akshaybhalotia** [brave/brave-browser#17359]
## Description 

When pasting a link in new tab and hitting enter (or clicking the URL suggestion below the address bar), the tab starts loading but goes blank. Repeating the above steps loads the page fine.

## Steps to Reproduce

   1. Open new tab.
   2. Paste address and press enter.

## Actual result:

Address bar becomes blank.

## Expected result:

Page loads

## Reproduces how often: 
Easily reproduced

## Brave version (brave://version info)

Brave | 1.26.74 Chromium: 91.0.4472.124 (Official Build) (x86_64 translated)
-- | --
Revision | 7345a6d1bfcaff81162a957e9b7d52649fe2ac38-refs/branch-heads/4472_114@{#6}
OS | macOS Version 12.0 (Build 21A5294g)
JavaScript | V8 9.1.269.36
User agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36
Command Line | /Applications/Brave Browser.app/Contents/MacOS/Brave Browser --enable-dom-distiller --disable-domain-reliability --no-pings --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --variations-server-url=https://variations.brave.com/seed --enable-features=WebUIDarkMode,PrefetchPrivacyChanges,PasswordImport,ReducedReferrerGranularity,LegacyTLSEnforced,AutoupgradeMixedContent,SafetyTip --disable-features=FederatedLearningOfCohorts,AutofillEnableAccountWalletStorage,FledgeInterestGroupAPI,EnableProfilePickerOnStartup,TextFragmentAnchor,TrustTokens,DirectSockets,SubresourceWebBundles,NetworkTimeServiceQuerying,LangClientHintHeader,IdleDetection,SignedExchangePrefetchCacheForNavigations,AutofillServerCommunication,LiveCaption,SignedExchangeSubresourcePrefetch,InterestCohortAPIOriginTrial,FirstPartySets,NotificationTriggers,WebOTP,FlocIdComputedEventLogging,HandwritingRecognitionWebPlatformApiFinch,FledgeInterestGroups,InterestCohortFeaturePolicy,HandwritingRecognitionWebPlatformApi --flag-switches-begin --load-media-router-component-extension=1 --flag-switches-end --restore-last-session
Executable Path | /Applications/Brave Browser.app/Contents/MacOS/Brave Browser
Profile Path | /Users/akshaybhalotia/Library/Application Support/BraveSoftware/Brave-Browser/Default
Variations | AdRewardsStudy:NextPaymentDayEphemeralStorageStudy:EnabledMacCoreLocationBackendStudy:EnabledNativeCosmeticFilteringStudy:EnabledPermissionLifetimeReleaseStudy:Enabled

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Not sure
- Can you reproduce this issue with the nightly channel? Not sure

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? Not sure

**Comment by akshaybhalotia** [comment]
Link to video: https://drive.google.com/file/d/1O-3eERTj3GviN0brVZoh_LCFtnaScQ-8/view

**Comment by rebron** [comment]
Hi @akshaybhalotia are you able to reproduce this consistently? I tried your steps but actually followed your video where you opened up a private new window and it looks like it's working properly. Is the url that you are pasting redirecting you to a different page? That could be it?

**Comment by dentistformyeye** [comment]
I can't reproduce the bug using a new window, but I get a similar problem with new tabs. On macOS:
1. Have a tab already open. Make sure that no open tabs are new tab pages.
2. Press command-T.
3. Press a letter key. Press enter (to load a search page). This step must be completed before the new tab page finishes loading.

Whenever I open a new tab in the same window, there's a white flash. Step 3 must be completed before the white flash occurs. I think that when a new tab page gets opened, it reloads itself immediately, which clears the address bar.

When a new tab page is already open, the white flash occurs sooner for subsequent new tab pages; it seems impossible to complete the following step 3 before the white flash. The following steps successfully load a search page:
1. Have a new tab page already open.
2. Press command-T.
3. Press a letter key. Press enter. Complete this step as quickly as possible.


I'm on macOS 10.15.7.

**Comment by rebron** [comment]
@akshaybhalotia Is what you're seeing happening consistently?

**Comment by akshaybhalotia** [comment]
Yes, I am seeing this pretty consistently. Although after reading @dentistformyeye 's comment, I have made the same observation. It happens only if I do these steps _before_ the new tab loads.

**Comment by oysteins-a** [comment]
I'm having the same issue on Windows 10 for several releases now. If I type and hit enter in the address bar before the new tab page has loaded (~2 seconds) it gets cleared.
This also occurs in Nightly.
If the new tab page would load immediately, this wouldn't be an issue.

brave://version info:
<html>
<body>


Brave | 1.28.105 Chromium: 92.0.4515.131 (Official Build) (64-bit)
-- | --
Revision | 6b8d6c56ce21e38a72f7c4becb5abc1fa5134f29-refs/branch-heads/4515@{#1933}
OS | Windows 10 OS Version 2009 (Build 19043.1165)
JavaScript | V8 9.2.230.22
User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
Command Line | "C:\Program Files\BraveSoftware\Brave-Browser\Application\brave.exe" --enable-dom-distiller --disable-domain-reliability --no-pings --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --variations-server-url=https://variations.brave.com/seed --enable-features=PrefetchPrivacyChanges,LegacyTLSEnforced,WinrtGeolocationImplementation,SafetyTip,AutoupgradeMixedContent,WebUIDarkMode,PasswordImport,ReducedReferrerGranularity --disable-features=InterestCohortFeaturePolicy,HandwritingRecognitionWebPlatformApi,FledgeInterestGroups,LiveCaption,AutofillServerCommunication,IdleDetection,FlocIdComputedEventLogging,HandwritingRecognitionWebPlatformApiFinch,AutofillEnableAccountWalletStorage,FledgeInterestGroupAPI,EnableProfilePickerOnStartup,TextFragmentAnchor,PrivacySandboxSettings,FederatedLearningOfCohorts,WebOTP,NotificationTriggers,LangClientHintHeader,SubresourceWebBundles,NetworkTimeServiceQuerying,SignedExchangePrefetchCacheForNavigations,SignedExchangeSubresourcePrefetch,DirectSockets,TrustTokens,InterestCohortAPIOriginTrial,FirstPartySets,EnablePasswordsAccountStorage --flag-switches-begin --flag-switches-end
Executable Path | C:\Program Files\BraveSoftware\Brave-Browser\Application\brave.exe
Profile Path | C:\Users\Øystein\AppData\Local\BraveSoftware\Brave-Browser\User Data\Default
Variations | AdRewardsStudy:NextPaymentDayEphemeralStorageStudy:EnabledNativeCosmeticFilteringStudy:EnabledPermissionLifetimeReleaseStudy:EnabledSpeedreaderReleaseStudy:Disabled


</body>
</html>
